policyengine_us/parameters/gov/contrib/reconciliation/ctc/one_person_ssn_req.yaml

```
@@ -0,0 +1,12 @@
+description: The Committee on Ways and Means proposed to limit the CTC to households with at least one person with an SSN, which comes in effect if this is true. 
```

No acronyms in descriptions please. Could you also check the source is Senate?
```suggestion
description: The Senate Finance Committee proposed to limit the Child Tax Credit to households with at least one person with a Social Security number, which comes in effect if this is true. 
```